project-management/Sprint_Reports/Sprint_2.md

```diff
@@ -0,0 +1,111 @@
+* [`#176`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/176): Document the process of how we will get feedback from the community
+* [`#177`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/177): Create a retrospective form
+* [`#158`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/158): Alan Turing feedback on Demo
+* [`#173`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/173): Develop paired down secured engagement component
+* [`#174`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/174): Develop 'add your experience' page component
+* [`#175`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/175): Decide on landing page strapline
+* [`#172`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/172): Develop landing page footer
+* [`#171`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/171): Develop hamburger menu component
+* [`#170`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/170): Develop layout implementation of the landing page Hero/Jumbotron component
+* [`#169`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/169): Develop layout of landing page function boxes
+* [`#168`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/168): Implement wireframe landing page
+* [`#167`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/167): Implementation of landing page header
+* [`#166`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/166): Translateable platform
+* [`#163`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/163): List of questions for community
+* [`#160`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/160): Documentation for user stories
+* [`#145`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/145): Functional testing guidelines
+* [`#148`](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/148): Turn user stories from google docs into issues on Github
+
+### Challenges
+
+The team met the following challenges or blockers during this sprint.
+
+* We have struggled to gain timely feedback from the community
+  * Related to issues: [#174](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/174), [#158](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/158)
+  * we now have a new member (Paul Crayson) in the Fujitsu team with autistic contacts from his personal experiences. 
+* We have struggled to get feedback from Autistica
+  * Related to issues: [#120](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/174), [#158](https://github.com/alan-turing-institute/AutisticaCitizenScience/issues/120)
```

Comment:
I think this phrasing is unkind wrt two points: 1) we haven't invited feedback from the community - there was no sprint report for them to review and no promotion to indicate what feedback we were looking for, and 2) there is a global pandemic and Autistica are under extreme stress both from a loss of financial support and an increase in demand for their support.

Please rephrase.